pyproject.toml

```diff
@@ -60,3 +61,6 @@ line_length = 140
 [build-system]
 requires = [ "poetry-core>=1.0.0" ]
 build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.extras]
+mlmodel = ["numpy"]
```

Comment:
I know we agreed on mlmodel earlier, but given we can't hide the directory anyway, I think numpy would be a better name given it'll be more generic. @njooma thoughts?

Reply:
I think it should still be `mlmodel` because it should be the name of the enabled feature, not its dependencies. No we can't hide it, but we do not allow it to be imported without its dependencies. Plus, this gives us the option to add more dependencies to that feature

Reply:
aight, my feelings aren't too strong, so this is fine for now